### DISCUSSION TITLE
How to add MkDocs to GitHub pages

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#21794] [Other Feature Feedback, Questions, & Ideas]
<p>I would like to host my API documentation through GitHub Pages and by using MkDocs with the Material theme.</p>
<p>But all the docs (including the ones from MkDocs themself) I found are too confusing for me.</p>
<p>So I would appreciate if anyone here would have an Idiot proof instruction on how you could set pages up using MkDocs, preferably without needing to install any software/tools on my own PC.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/andre601">@andre601</a>,</p>
<p>Thanks for being here! A community member created a GitHub Action to automate this, might be worth checking out for what you are trying to accomplish.</p>
<p>- <a href="https://github.com/mhausenblas/mkdocs-deploy-gh-pages" rel="nofollow noopener">https://github.com/mhausenblas/mkdocs-deploy-gh-pages</a></p>
